command/bridge/server/server.go

```diff
@@ -68,6 +69,20 @@ func setFlags(cmd *cobra.Command) {
 		false,
 		"use the official geth image instead of the console fork",
 	)
+
+	cmd.Flags().Uint64Var(
+		&params.chainID,
+		"chain-id",
+		666,
```

Comment:
```suggestion
		101,
```

And move it to a constant.